includes/class-sensei.php

```diff
@@ -4,7 +4,7 @@
 } // End if().
 
 // Load Usage Tracking
-require_once( dirname( __FILE__ ) . '/../lib/usage-tracking/class-usage-tracking.php' );
+require_once( __DIR__ . '/../lib/usage-tracking/class-usage-tracking.php' );
```

Comment:
`__DIR__` is PHP 5.3+, just in case Sensei still advertises 5.2.

Reply:
Ahh. Darn. As far as I know, we support 5.2, since that's what WP supports. Reverted this.